test/src/test/resources/file.conf

```diff
@@ -46,8 +46,12 @@ transport {
   }
   serialization = "seata"
   compressor = "none"
+
```

Comment:
file.conf default value also to 'true',  ensure consistency

Reply:
Will cause unit tests to fail